api/ceramic.yaml

```diff
@@ -632,3 +679,33 @@ components:
         name:
           type: string
           description: Name of the Ceramic network
+    StreamState:
+      title: State of a Ceramic stream
+      description: The state of a Ceramic stream as defined by the stream type aggregation and conflict resolution rules.
+      type: object
+      required:
+        - id
+        - event_cid
```

Comment:
historically we've called this `tip`, do we want to maintain that convention?

Reply:
No sure, I like the name event_cid because that makes it clear you can use the cid to lookup an event directly. With tip it's not clear what the CID points to